content/docs/handling-events.md

```diff
@@ -54,11 +54,11 @@ function ActionLink() {
 }
 ```
 
-Here, `e` is a synthetic event. React defines these synthetic events according to the [W3C spec](https://www.w3.org/TR/DOM-Level-3-Events/), so you don't need to worry about cross-browser compatibility. See the [`SyntheticEvent`](/docs/events.html) reference guide to learn more.
+यहाँ पर 'e' एक synthetic इवेंट है | React इन synthetic इवेंट्स को [W3C spec](https://www.w3.org/TR/DOM-Level-3-Events/) के हिसाब से परिभाषित करता है, इसी लिए आपको cross-browser compatibility कि चिंता नही करनी चाहिए | और सिखने के लिए [`SyntheticEvent`](/docs/events.html) कि reference guide को देखें |
```

Comment:
```
'e'
```
=>
```
`e`
```

Reply:
`synthetic` => `सिंथेटिक`

Reply:
> `synthetic` => `सिंथेटिक`

You missed this

Reply:
This has also been resolved, am not changing 
```
[`SyntheticEvent`](/docs/events.html)
```
If you think I should then tell me.